src/com/rcv/JsonParser.java

```diff
@@ -22,6 +23,7 @@
   // file access: read
   // returns: instance of the object parsed from json or null if there was a problem
   static <T> T parseObjectFromFile(String jsonFilePath, Class<T> valueType) {
+    // TODO: consolidate two return statements in this method into one?
```

Comment:
oh, yeah, please fix that while you're here, if you don't mind

Reply:
Yep, done in the next PR.